### DISCUSSION TITLE
Statamic doesn't appear to track asset .meta files?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#8009] [General]
Hi

Statamic doesn't appear to track asset `.yaml` files inside `.meta`? Is it because the meta directory is hidden?

I updated the meta for an image, saved changes but they're not committed to git?

```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   public/assets/images/.meta/support-group-1.jpg.yaml
 ```
 
 Updates to entries are committed and pushed as expected.
 
 Is this a known issue? Or is there an additional bit of config required to get this work?

gitignore
```
/node_modules
/public/hot
/public/storage
/public/vendor/statamic
/storage/*.key
/vendor
.env
.env.backup
.phpunit.result.cache
.php_cs.cache
docker-compose.override.yml
Homestead.json
Homestead.yaml
npm-debug.log
yarn-error.log
.DS_Store
/.idea
/.vscode
```

```
Statamic
Addons: 1
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 3.4.8 PRO
```

**Comment by duncanmcclean** [comment]
Do you have any `.gitignore` files in any of the folders in `public` that could be ignoring it?

**Reply by stephenmeehanuk** [reply]
Hi,

I've not made any changes to `.gitignore` files since installation.

My installation of Statamic seems to ignore files in `.meta` directories?

Is this not a common thing?

**Comment by jasonvarga** [comment]
It looks like Git itself is tracking them, since you've shown that in the output:

```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   public/assets/images/.meta/support-group-1.jpg.yaml
```

But the Git integration in Statamic is not committing/pushing them because you haven't configured it to commit files in that directory.

You can add `public_path('assets')` to the `paths` array in `config/statamic/git.php`.

```diff
    'paths' => [
        base_path('content'),
        base_path('users'),
        resource_path('blueprints'),
        resource_path('fieldsets'),
        resource_path('forms'),
        resource_path('users'),
        storage_path('forms'),
+       public_path('assets'),
    ],
```

**Reply by stephenmeehanuk** [reply]
Thanks! I can now pull `.meta` updates into my local site 

Wondering, why this isn't enabled by default?

**Reply by jasonvarga** [reply]
People often have so many assets they prefer not to track them. But of course you're welcome to.

**Reply by stephenmeehanuk** [reply]
Ah, ok. I don't track the assets in Git, but I did want to track the .meta files.

Thanks again.
